### DISCUSSION TITLE
hELLO 3.0 버그 + 건의사항

### DISCUSSION THREAD

**Discussion by jhhan128** [pronist/hELLO#92] [Ideas]
제가 경험한 버그들입니다. 혹시 더 발견되면 추가하겠습니다.

![IMG_AB2A1F9524FB-1](https://user-images.githubusercontent.com/54146988/93575753-fa8aba00-f9d4-11ea-95f7-b15699de4a59.jpeg)
모바일에서 카테고리에 들어가면 제목이 저렇게 표시됩니다. 사파리도, 크롬도 마찬가지였습니다.

![IMG_0074](https://user-images.githubusercontent.com/54146988/93576156-84d31e00-f9d5-11ea-8c61-785a0b33bb44.jpeg)
또한, 커버 형식 **리스트**에서 구분선이 다크 모드 일때도 까맣게 나와 잘 보이지 않습니다.

![IMG_0075](https://user-images.githubusercontent.com/54146988/93576420-d380b800-f9d5-11ea-8890-39c1929c7a12.jpeg)
전체 글 보기에서 블로그 이름이 다음과 같이 표시되네요. 특수문자라 그런걸까요?

---

![IMG_0076](https://user-images.githubusercontent.com/54146988/93577009-83562580-f9d6-11ea-88d4-36a42007852b.jpeg)
커버에 Magazine 스킨의 **슬라이드 리스트 2** 같은 형태의 커버가 들어가면 좋을 것 같습니다.
그리고 Poster스킨의 커버처럼 그냥 사진만 나타내는 커버도 있으면 좋을 것 같습니다.

또한, 사이드바가 표시되는 최소 가로폭을 1400px로 바꾸는 것이 좋을 것 같습니다. 13인치 맥북에서 사이드바가 나올 공간은 충분한데, 표시되지 않을 필요가 없을 것 같습니다.

---

빨리 수정해주시면 감사하겠습니다~

**Comment by pronist** [comment]
@jhhan128 버그를 찾아주셔서 감사합니다. 버그수정은 대체로 이루어질 예정이나 개선의견이나 추가요청은 검토를 거친뒤 결정하므로 반영되지 않을 수도 있습니다 :)

**Comment by pronist** [comment]
반영 내역을 알려드립니다.

1. 제목이 겹치는 현상 -> 반영
2. 리스트 다크모드 라인 -> 반영
3. 블로그 타이틀 특수문자 -> **티스토리에서 그렇게 바꿔주는 것이므로** 반영x
4. 커버 -> **갤러리 커버와 역할이 겹치므로** 반영x
5. 사이드바 기준 1400px -> 반영

버그 신고 및 개선의견 요청에 감사드립니다 :)
